src/libs/ReportUtils.js

```diff
@@ -1250,6 +1224,33 @@ function getMoneyRequestReportName(report, policy = undefined) {
     return payerPaidAmountMesssage;
 }
 
+/**
+ * Get the report for a reportID
+ *
+ * @param {String} reportID
+ * @returns {Object}
+ */
+function getReport(reportID) {
+    return lodashGet(allReports, `${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {});
+}
+
+/**
+ * Gets transaction amount, currency and comment
+ *
+ * @param {Object} transaction
+ * @returns {Object}
+ */
+function getTransactionDetails(transaction) {
+    const reportID = transaction.reportID;
+    const report = getReport(reportID);
```

Comment:
Its not reused so I think we dont need to assign it to new variable
```suggestion
    const report = getReport(transaction.reportID);
```